### PR TITLE
scripts/api-generator: add _id field for each object

### PR DESCRIPTION
**Pull request by rjeczalik**

### CODE REVIEW

File: scripts/api-generator.coffee [hunk 1]
```diff
@@ -132,7 +132,7 @@ generateDefinition = (model) ->
 
   schema = model.describeSchema()
   def    = { type: 'object' }
-  props  = {}
+  props  = { _id: { type: 'string' } }
```
Comment (rjeczalik) [inline scripts/api-generator.coffee]: /cc @gokmen

**Review by gokmen — APPROVED** [review]

**Comment by gokmen** [comment]
you need to update swagger.json it seems

**Comment by rjeczalik** [comment]
Well I did 😕 

```
$ ./run exec ./scripts/api-generator.coffee
Swagger.json updated succesfully!
$ git diff
$
```

**Comment by rjeczalik** [comment]
Ok, apparently different node version - the 6 adds `default` field. I think it's a bug - swagger spec does not list a `default` type. 🤔

**Comment by gokmen** [comment]
that can be related with api-generator, I'll take a look at it asap.

**Comment by rjeczalik** [comment]
I made some shotgun surgery with https://github.com/koding/koding/pull/10284/commits/cdf9296c6a62c663cb0fb47488ae9cd45f570f9c and the default:default objects are gone.

@gokmen wdyt?

File: scripts/api-generator.coffee [hunk 2]
```diff
@@ -135,10 +138,12 @@ generateDefinition = (model) ->
   props  = { _id: { type: 'string' } }
 
   for field, prop of schema
+    continue if field is 'default' and prop.type is 'default'
 
     if 'type' not in Object.keys prop
       props[field] = { properties: {} }
       for subfield, subprop of prop
+        continue if subfield is 'default' and subprop.type is 'default'
         props[field].properties[subfield] = getProps subprop, def, field
```
Comment (rjeczalik) [inline scripts/api-generator.coffee]: Shouldn't this line be `getProps subprop, def, subfield`?

**Comment by gokmen** [comment]
@rjeczalik for now let's merge this as is if it passes tests, then I'll take a look at the https://github.com/koding/koding/pull/10284#discussion_r95657628 and inform you, ok?

**Comment by rjeczalik** [comment]
sounds good

**Comment by rjeczalik** [comment]
@cihangir ptal when you have chance

**Review by cihangir — COMMENTED** [review]
👌

**Pull request merged**